config/initializers/delayed_job_config.rb

```diff
@@ -1,5 +1,5 @@
 Delayed::Worker.destroy_failed_jobs = false
-Delayed::Worker.sleep_delay = 0
+Delayed::Worker.sleep_delay = 0.5
```

Comment:
This was causing 100% CPU usage, as we were constantly polling the database. 0.5 is still somewhat fast, and we'd probably be better off making the shorter lived tasks synchronous.

Given we already have redis available, is might be worth switching to sidkiq for our job management, as its generally less resource intensive.